python/pyspark/ml/stat/distribution.py

```diff
@@ -0,0 +1,267 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from pyspark.ml.linalg import DenseVector, DenseMatrix, Vector
+import numpy as np
+
+__all__ = ['MultivariateGaussian']
+
+
+
+class MultivariateGaussian():
+    """
+    This class provides basic functionality for a Multivariate Gaussian (Normal) Distribution. In
+     the event that the covariance matrix is singular, the density will be computed in a
+    reduced dimensional subspace under which the distribution is supported.
+    (see [[http://en.wikipedia.org/wiki/Multivariate_normal_distribution#Degenerate_case]])
+
+    mu The mean vector of the distribution
+    sigma The covariance matrix of the distribution
+
+
+    >>> mu = Vectors.dense([0.0, 0.0])
```

Comment:
Vectors is used for creating Vector Object. But this class takes already created object as input.